frappe/translate.py

```diff
@@ -159,63 +159,6 @@ def get_lang_dict():
 	)
 
 
-def get_dict(fortype: str, name: str | None = None) -> dict[str, str]:
-	"""Returns translation dict for a type of object.
-
-	:param fortype: must be one of `doctype`, `page`, `report`, `include`, `jsfile`, `boot`
-	:param name: name of the document for which assets are to be returned.
-	"""
-	fortype = fortype.lower()
-	asset_key = fortype + ":" + (name or "-")
-	translation_assets = frappe.cache.hget("translation_assets", frappe.local.lang) or {}
-
-	if asset_key not in translation_assets:
-		messages = []
-		if fortype == "doctype":
-			messages = get_messages_from_doctype(name)
-		elif fortype == "page":
-			messages = get_messages_from_page(name)
-		elif fortype == "report":
-			messages = get_messages_from_report(name)
-		elif fortype == "include":
-			messages = get_messages_from_include_files()
-		elif fortype == "jsfile":
-			messages = get_messages_from_file(name)
-		elif fortype == "boot":
-			apps = frappe.get_all_apps(True)
-			for app in apps:
-				messages.extend(get_server_messages(app))
-
-			messages += get_messages_from_navbar()
-			messages += get_messages_from_include_files()
-			messages += (
-				frappe.qb.from_("Print Format").select(PseudoColumn("'Print Format:'"), "name")
-			).run()
-			messages += (frappe.qb.from_("DocType").select(PseudoColumn("'DocType:'"), "name")).run()
-			messages += frappe.qb.from_("Role").select(PseudoColumn("'Role:'"), "name").run()
-			messages += (frappe.qb.from_("Module Def").select(PseudoColumn("'Module:'"), "name")).run()
-			messages += (
-				frappe.qb.from_("Workspace Shortcut")
-				.where(Field("format").isnotnull())
-				.select(PseudoColumn("''"), "format")
-			).run()
-			messages += (frappe.qb.from_("Onboarding Step").select(PseudoColumn("''"), "title")).run()
-
-		messages = deduplicate_messages(messages)
-		message_dict = make_dict_from_messages(messages, load_user_translation=False)
-		message_dict.update(get_dict_from_hooks(fortype, name))
```

Comment:
Everything else LGTM.

Reply:
Thanks, I'll check this

Reply:
The `get_translated_dict` hook only ever pointed to `frappe.geo.country_info.get_translated_dict`, which returns  translations for timezones, countries and currencies (provided by babel).

I removed the hook and included these translations in `get_all_translations`.

Reply:
But I'm not sure if we need translations of currency codes and timezones at all. If in doubt, I'd prefer to ditch them.

Reply:
I don't think we need them too. 😅 

Change in money_in_words might not be acceptable too. Currency symbol is perfectly fine instead of full currency name there.

Reply:
I removed the translations of currency codes and timezones. Translations for country names remain available everywhere.
The output of `money_in_words` should stay unchanged now - I reverted my changes to the respective test.